playbooks/symfony/symfony-messenger.md

```diff
@@ -0,0 +1,30 @@
+---
+title: When to use a Symfony Messenger component 🚚
+description: A brief guideline on when is the right time to use a Symfony Messenger component
+---
+
+Symfony Messenger is a very powerful component that allows you to implement some level of concurrency in your
+PHP/Symfony projects that are usually non-concurrent.
+
+More info about it can be found in
+[official documentation.](https://symfony.com/doc/current/components/messenger.html) A really nice tutorial can ba
+found on [Symfonycasts.](https://symfonycasts.com/screencast/messenger) And we also have an internal education
+recorded from Week #48 (if you don't know where to find it, contact your nearest team-lead).  
+If you are wondering how to notify the client when concurrent job has been finished, take a look at  
+the [Mercure protocol and Symfony Mercure component](https://symfony.com/doc/current/mercure.html)
+
+### When to use the Messenger component?
+
+If you are here because you want to know should you use the Messenger component, go through the cases below, and if
```

Comment:
... want to know if you should use ...
```suggestion
If you are here because you want to know if you should  use the Messenger component, go through the cases below, and if
```